builds.yml

```diff
@@ -198,9 +198,6 @@ env:
   # This variable is read by Trezor's source and breaks build if not included
   - ASSET_PREFIX: null
 
-  # This variable is needed for MMI to have it's specific Sentry Key
```

Comment:
Is it safe to remove this  variable?

Reply:
Yes, we dont need it there, its down bellow in our build declaration.